doc/stake-pool-operations/5_register_key.md

```diff
@@ -25,11 +25,16 @@ For the transaction draft, --tx.out, --invalid-hereafter and --fee can be set to
 
 #### Calculate fees
 
+Compute the minimum fee.
+
+Note that only one witness is required because
+[Chapter 3.3 "we do not require a witness to register a stake address"](https://hydra.iohk.io/build/6752481/download/1/delegation_design_spec.pdf).
```

Comment:
@newhoggy I believe this spec link needs to be updated.

Reply:
Need a way to follow this up: https://input-output-rnd.slack.com/archives/C02H2Q4L54Y/p1679055599289319?thread_ts=1678354118.254929&cid=C02H2Q4L54Y

Reply:
This is the link https://github.com/input-output-hk/cardano-ledger/releases/latest/download/shelley-delegation.pdf

Reply:
Fixed!